### ISSUE TITLE
No Portal upon world creation

### ISSUE THREAD

**Issue by Itadiki** [davqvist/Exoria#110]
Tried couple of world generation and found two seed than doesn t have a portal upon creation
Seeds:
-8278398733869670552
-8526632756617632963

**Comment by davqvist** [comment]
Happened to me rarely as well, but not sure what I can do about that. I will do some more investigation

**Comment by Pandemoneus** [comment]
I have that happen in every world I created so far. Is there supposed to be a portal generating?

[edit]
Alright, and in the next 3 seeds I tried I got a portal each time.

**Comment by Ladysarajane** [comment]
In one seed, I obtained 3 portals,  3 root places and so far 1 botania house. The 3 root places are within site of each other. Portals only show for me 3 out of 10 worlds I create.

**Comment by Beraias** [comment]
I have this same issue. No portal spawned in.

**Comment by davqvist** [comment]
Yeah not sure why exactly, but luckily you can just make a new world.

**Comment by wormzjl** [comment]
Sometimes other ruins does not spawn as well
